Description: Add alleg4 manpage suffix
 Add alleg4 suffix to manual page file names to prevent
 possible name clashes.
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -159,7 +159,7 @@
 set(MANDIR "man" CACHE STRING "Install man pages into this directory")
 
 if(WANT_DOCS_MAN)
-    set(man_out ${CMAKE_CURRENT_BINARY_DIR}/man/allegro_init.3)
+    set(man_out ${CMAKE_CURRENT_BINARY_DIR}/man/allegro_init.3alleg4)
     set(man_src ${CMAKE_CURRENT_SOURCE_DIR}/src/allegro._tx)
 
     add_custom_command(
@@ -172,7 +172,7 @@
     # I'm assuming only Unix users want man pages installed, but djgpp supports
     # man pages to so this might need to change.
     if(UNIX)
-        file(GLOB MAN_PAGES ${CMAKE_CURRENT_BINARY_DIR}/man/*.3)
+        file(GLOB MAN_PAGES ${CMAKE_CURRENT_BINARY_DIR}/man/*.3alleg4)
         install(FILES ${MAN_PAGES}
                 DESTINATION ${MANDIR}/man3
                 )
--- a/docs/src/makedoc/makedoc.c
+++ b/docs/src/makedoc/makedoc.c
@@ -160,7 +160,7 @@
       printf("\t-htm[l] filename.htm[l]\n");
       printf("\t-rtf filename.rtf\n");
       printf("\t-t[e]xi filename.t[e]xi\n");
-      printf("\t-man filename.3\n");
+      printf("\t-man filename.3alleg4\n");
       printf("\t-chm filename.htm[l]\n");
       printf("\t-devhelp filename.htm[l]\n");
       printf("\t-scite filename.api\n");
--- a/docs/src/makedoc._tx
+++ b/docs/src/makedoc._tx
@@ -49,7 +49,7 @@
    modify the makedoc tool. This output is generated by default and available
    on all supported platforms.
 <li>
-   `.3'<br>
+   `.3alleg4'<br>
    Unix man pages for the section "Library calls (functions within program
    libraries)". As usual these man pages will contain a synopsis,
    description and see also sections. Certain parts of the documentation
--- a/docs/src/makedoc/makeman.c
+++ b/docs/src/makedoc/makeman.c
@@ -172,7 +172,7 @@
 			if (!f2)
 			   return 1;
 
-			fprintf(f2, ".so man%s/%s\n", get_extension(filename), get_filename(buf));
+			fprintf(f2, ".so man%1.1s/%s\n", get_extension(filename), get_filename(buf));
 			fclose(f2);
 		     }
 
